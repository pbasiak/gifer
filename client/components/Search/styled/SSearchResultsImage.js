import styled from 'styled-components';

export const SSearchResultsImage = styled.img`
    margin: 5px;
    max-width: 100%;
    cursor: pointer;
`;
