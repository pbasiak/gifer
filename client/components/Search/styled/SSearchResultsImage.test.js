import React from 'react';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';
import 'jest-styled-components';
import { SSearchResultsImage } from './SSearchResultsImage';

it('should match snapshot', () => {
    const wrapper = shallow(<SSearchResultsImage />);
    expect(toJson(wrapper)).toMatchSnapshot();
});