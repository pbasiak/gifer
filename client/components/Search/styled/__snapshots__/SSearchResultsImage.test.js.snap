// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
    margin: 5px;
    max-width: 100%;
    cursor: pointer;
",
        ],
      },
      "displayName": "styled.img",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "img",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
